spec/models/openid.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe Openid do
+  subject { create(:openid) }
+
+  describe 'validation' do
+    it { should validate_presence_of(:provider) }
```

Comment:
Please use is_expected.to instead of should

Reply:
Okay